Libraries/LibC/stdlib.h

```diff
@@ -41,13 +41,13 @@ __BEGIN_DECLS
 __attribute__((malloc)) __attribute__((alloc_size(1))) void* malloc(size_t);
 __attribute__((malloc)) __attribute__((alloc_size(1, 2))) void* calloc(size_t nmemb, size_t);
 size_t malloc_size(void*);
-void serenity_dump_malloc_stats(void);
+void serenity_dump_malloc_stats();
 void free(void*);
 __attribute__((alloc_size(2))) void* realloc(void* ptr, size_t);
 char* getenv(const char* name);
 int putenv(char*);
 int unsetenv(const char*);
-int clearenv(void);
+int clearenv();
```

Comment:
It's incorrect to do this to C headers

In C, `void foo()` declares a function *without specifying anything* about its parameter count / types (somewhat like `void foo(...)`, but different); while `void foo(void)` declares a function taking no arguments.

Reply:
Oops! You are right. I didn't think about these being used in C code, but there is no reason they shouldn't be. I just opened PR #4522 to address this. Thanks for catching it.